### PR TITLE
Release Candidate version 2.6.3

### PR DESCRIPTION
**Pull request by aburrell**
# Description

Fixes a memory leak in the C wrapper for array lat/lon conversion, improves the documentation, and updates the unit testing.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

# How Has This Been Tested?

Ran the new examples in the docs, added more unit tests, and ensured `aacgmv2.convert_latlon_arr` runs faster than `aacgmv2.convert_latlon` for multiple locations.

**Test Configuration**:
* Operating system: OS X Big Sur
* Version number: Python 3.9
* Any details about your local setup that are relevant

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``Changelog.rst``, summarising the changes
- [x] Add yourself to ``AUTHORS.rst``

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -93,14 +97,3 @@ For merging, you should:
        have trouble building all the testing environments, you can rely on
        Travis and AppVeyor - they will run the tests for each change you add in
```
Comment (aburrell) [inline CONTRIBUTING.rst:98]:
```suggestion
       GitHub Actions - it will run the tests for each change you add in
```

**Comment by aburrell** [comment]
@jklenzing, @mattkjames7, @mshumko, @JouleCai, @dchaddock, @billetd you are all involved in packages that depend on aacgmv2.  This is just a heads up that I plan on releasing a new version next week, barring any complications. If you'd like to take a poke at this to ensure there are no issues with your packages, there is a release candidate up on testPyPi: https://test.pypi.org/project/aacgmv2/2.6.3/

**Comment by dchaddock** [comment]
Hi Angeline - things look good from my tests. Thanks for the heads up!

**Comment by carleyjmartin** [comment]
Looking all good for us at pyDARN, thanks!

**Comment by jklenzing** [comment]
It works for pysatMissions using a local install from github.  Having issues with the install from test pypi though (both locally and at github actions).

**Comment by aburrell** [comment]
> It works for pysatMissions using a local install from github. Having issues with the install from test pypi though (both locally and at github actions).

The test PyPi install works for me locally, but testPyPi does throw a lot of connection warnings.  Could be related to that, keep me posted.

**Comment by mshumko** [comment]
Thanks for the heads up! aurora-asi-lib works.

P.S. If anyone got a pip install error where pip did not find `setuptools` (even if is already installed), you need to specify the `--extra-index-url`. In other words:

```bash
python3 -m pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ aacgmv2==2.6.3
```

**Comment by mshumko** [comment]
@jklenzing of course!

**Comment by JouleCai** [comment]
> Thanks for the heads up! aurora-asi-lib works.
> 
> P.S. If anyone got a pip install error where pip did not find `setuptools` (even if is already installed), you need to specify the `--extra-index-url`. In other words:
> 
> ```shell
> python3 -m pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple/ aacgmv2==2.6.3
> ```

Thanks a lot! The new version works for GeospaceLAB.

**Comment by mattkjames7** [comment]
Thanks for the update, seems to work as expected with my code!

**Pull request merged**